server/src/routes/pointsFetch.ts

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const path = require("path");
+const app = express();
+import {Response} from "express";
+
+app.get('/api/vi/geojson', (res: Response) => {
```

Comment:
v1 not vi.
also the routes do not encapsulate from the start. 
you are creating another express app here. 
the flow of the program is not correct.